app/components/ui/molecules/StaffGrid.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import styled from 'styled-components'
+import StaffCard from './StaffCard'
+
+const { Flex, Box } = require('grid-styled')
+
+const Grid = styled(Flex)`
+  flex-flow: row wrap;
+`
+const Card = styled(Box)`
+  padding-top: 20px;
```

Comment:
Unless we have specific designs for this, I'd suggest using `24px` here (i.e. `space.3`) for consistency with the grid.